sampleSC/pom.xml

```diff
@@ -28,6 +28,28 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>com.flipkart.poseidon</groupId>
+                <artifactId>cadfael-maven-plugin</artifactId>
+                <version>${project.parent.version}</version>
+                <configuration>
+                    <repository>http://10.85.59.116/artifactory/v1.0/artifacts/libs-release-local</repository>
```

Comment:
We can't have this here. Can't we use clojars?

Reply:
forgot to push this change, defining repositories like this is not supported anymore.